10-JavaScript/solutions/ruby-to-js-worksheet-solution.md

```diff
@@ -0,0 +1,246 @@
+# Ruby to JavaScript Worksheet
+Read the code in each section, then write the equivalent JavaScript code for the Ruby that is given. Start by writing it out in a text editor or on a piece of paper. Then try running it and tweak your code until it successfully runs with expected output.
+
+Each problem stands alone. Variables from previous problems do not exist.
+
+## Problem Set
+1. Ruby
+    ```ruby
+    ada_age = 2
+
+    if person_age < ada_age
+       print "This person is younger"
+    elsif ada_age < person_age
+       print "Ada is younger"
+    else
+       print "They’re the same!"
+    end
+    ```
+
+    <details>
+    <summary>
+    Javascript
+    </summary>
+
+    ```javascript
+    const adaAge = 2
+    let personAge = 35
+
+    if (personAge < adaAge){
+      console.log("This person is younger")
+    }
+    else if (adaAge < personAge){
+      console.log("Ada is younger")
+    }
+    else{
+      console.log("They’re the same!")
+    }
+    ```
+
+    </details>
+
+1. Ruby
+    ```ruby
+    x = 7
+    y = 7
+
+    if x > y || x == y
+       if x > y
+          print "x is bigger"
+       else
+          print "x = y"
+       end
+    else
+       print "y is bigger"
+    end
+    ```
+    <details>
+    <summary>
+    Javascript
+    </summary>
+
+    ```javascript
+    let x = 7
+    let y = 7
+
+    if (x > y || x == y){
+      if (x > y){
+        console.log("x is bigger")
+      }
+      else{
+        console.log("x = y")
+      }
+    }
+    else{
+      console.log("y is bigger")
+    }
```

Comment:
Minor style suggestion for things like this.  Putting a space before the { is most common.

```suggestion
    if (x > y || x == y) {
      if (x > y) {
        console.log("x is bigger")
      }
      else {
        console.log("x = y")
      }
    }
    else {
      console.log("y is bigger")
    }
```